src/physics/arcade/ArcadePhysics.js

```diff
@@ -145,31 +146,20 @@ var ArcadePhysics = new Class({
     },
 
     /**
-     * Checks for overlaps between two Game Objects. The objects can be any Game Object that have an Arcade Physics Body.
-     *
-     * Unlike {@link #collide} the objects are NOT automatically separated or have any physics applied, they merely test for overlap results.
-     *
-     * Both the first and second parameter can be arrays of objects, of differing types.
-     * If two arrays are passed, the contents of the first parameter will be tested against all contents of the 2nd parameter.
-     *
-     * ##### Tilemaps
-     *
-     * Any overlapping tiles, including blank/null tiles, will give a positive result. Tiles marked via {@link Phaser.Tilemap#setCollision} (and similar methods) have no special status, and callbacks added via {@link Phaser.Tilemap#setTileIndexCallback} or {@link Phaser.Tilemap#setTileLocationCallback} are not invoked. So calling this method without any callbacks isn't very useful.
-     *
-     * If you're interested only in whether an object overlaps a certain tile or class of tiles, filter the tiles with `processCallback` and then use the result returned by this method. Blank/null tiles can be excluded by their {@link Phaser.Tile#index index} (-1).
-     *
-     * If you want to take action on certain overlaps, examine the tiles in `collideCallback` and then handle as you like.
+     * Tests if Game Objects overlap. See {@link Phaser.Physics.Arcade.World#overlap}
```

Comment:
How come so much of this was removed?

Reply:
Yes. And I removed the _Tilemaps_ section because I'm not sure it's still accurate.